api/src/main.rs

```diff
@@ -1,10 +1,9 @@
 use config::Configuration;
+use dotenv::dotenv;
+use envconfig::Envconfig;
 use once_cell::sync::OnceCell;
 use std::process;
 use warp::Filter;
-extern crate log;
-use dotenv::dotenv;
-use envconfig::Envconfig;
```

Comment:
Sorting the imports is fine for me. Actually thought rustfmt would do this as well. They have something like this. We have to check that.

Reply:
I did it with `rustfmt`. It was just not sorted, cause of the `extern crate statements`.